hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyDeleteRequest.java

```diff
@@ -186,18 +191,20 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager, TermIn
         omClientResponse.setOmLockDetails(getOmLockDetails());
       }
     }
-
+    long endNanos = Time.monotonicNowNanos();
```

Comment:
I think this can be added within the `finally` block or you may also have separate `endNanos` for both success and failure latencies (can be added adjacent to `Result.Success`/`Result.Failure`).